setup.py

```diff
@@ -64,7 +64,7 @@ def _plugin_version_trim() -> str:
         f"dbt-postgres~={_plugin_version_trim()}",
         # dbt-redshift depends deeply on this package. it does not follow SemVer, therefore there have been breaking changes in previous patch releases
         # Pin to the patch or minor version, and bump in each new minor version of dbt-redshift.
-        "redshift-connector<2.0.918,>=2.0.913,!=2.0.914",
+        "redshift-connector<2.1.1,>=2.0.913,!=2.0.914",
```

Comment:
Is there a reason to pin less than a specific patch? I know we've been broken in the past, but if this package is indicating that they're going to start using minors, then I would think we pin to 2.1, not 2.1.x.

Reply:
we should follow up with them and see if that's actually what's happening here or if they just decided to do a minor version bump